client/components/social-buttons/google.js

```diff
@@ -106,6 +112,13 @@ class GoogleLoginButton extends Component {
 						this.props.responseHandler( currentUser, false );
 					}
 
+					if ( this.state.clickEvent ) {
+						// Make sure that handleClick Call happens on the next tick so that the popup always launches.
+						setTimeout( () => {
+							this.handleClick( this.state.clickEvent );
+						} );
```

Comment:
Why do we need the `setTimeout`? Reading the comment it's unclear what does "popup always launches" mean. What will break if I call `this.handleClick` right away?